app/models/pension/Journey.scala

```diff
@@ -46,7 +45,45 @@ object Journey {
       strBinder.unbind(key, journeyName.toString)
   }
 
+  private def pensionsSummaryRedirect(taxYear: Int): Result = Redirect(pensions.routes.PensionsSummaryController.show(taxYear))
+  private def overseasSummaryRedirect(taxYear: Int): Result = Redirect(pensions.routes.OverseasPensionsSummaryController.show(taxYear))
+  private def incomeFromPensionsSummaryRedirect(taxYear: Int): Result = Redirect(
+    pensions.incomeFromPensions.routes.IncomeFromPensionsSummaryController.show(taxYear))
```

Comment:
It might be nice for code organisation purposes to have a common locations module:

```scala
object AppLocations {
  val HOME: Int => Call =
    (taxYear: Int) => PensionsSummaryController.show(taxYear)

  val OVERSEAS_HOME: Int => Call =
    (taxYear: Int) => OverseasPensionsSummaryController.show(taxYear)

  val INCOME_FROM_PENSIONS_HOME: Int => Call =
    (taxYear: Int) => IncomeFromPensionsSummaryController.show(taxYear)

}
```
andThen...

```scala
object Journey {
 ....
case object AnnualAllowances extends Journey("annual-allowances") {
    def sectionCompletedRedirect(taxYear: Int): Result = Redirect(HOME(taxYear))
  }

case object OverseasPensionsSummary extends Journey("overseas-pensions-summary") {
    def sectionCompletedRedirect(taxYear: Int): Result = Redirect(OVERSEAS_HOME(taxYear))
  }
}
```

Reply:
That's much nicer, I'll do that